modules/metrics/src/test/java/org/apache/ignite/internal/metrics/sources/JvmMetricSourceTest.java

```diff
@@ -136,4 +157,49 @@ public ObjectName getObjectName() {
             throw new UnsupportedOperationException("Not supported in test implementation");
         }
     }
+
+    private static class GarbageCollectorBean implements GarbageCollectorMXBean {
```

Comment:
Maybe separate class?

Reply:
This is made by analogy with already existing internal class. Also, the class is only used here and the hosting file is not too long, so I'd prefer to leave it as is.

Reply:
Ok